public/video-ui/src/components/ReactApp.jsx

```diff
@@ -83,11 +83,11 @@ class ReactApp extends React.Component {
           query={this.props.location.query}
           error={this.props.error}
         />
-        {this.props.error
+        {this.props.error.message
           ? <div
-              key={this.props.errorKey}
+              key={this.props.error.key}
               className={`error-bar error-bar--animate`}
-              dangerouslySetInnerHTML={{ __html: this.props.error }}
+              dangerouslySetInnerHTML={{ __html: this.props.error.message }}
```

Comment:
Looking at the usages of `showError`, I don't see any instances of HTML being used for the error message (plain strings only) so we could render it directly - although might limit future use.

Reply:
I had the same thought, I can change it